html-tables/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>html-tables</title>
+</head>
+
+<body>
+  <h1>Student Grade Table</h1>
+  <table>
+    <thead>
+    <tr></tr>
+    <th scope="col">Student Name</th>
```

Comment:
you want to make sure that you are putting the `th` elements and `td` elements as children to the `tr`.